ingestion/src/metadata/test_suite/validations/column/base/columnValuesToBeNotInSet.py

```diff
@@ -0,0 +1,87 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# pylint: disable=invalid-name
+
+"""
+Validator for column value min to be between test case
+"""
+
+import traceback
+from abc import abstractmethod
+from ast import literal_eval
+from typing import Union
+
+from sqlalchemy import Column
+
+from metadata.generated.schema.tests.basic import (
+    TestCaseResult,
+    TestCaseStatus,
+    TestResultValue,
+)
+from metadata.orm_profiler.metrics.registry import Metrics
+from metadata.test_suite.validations.base_test_handler import BaseTestValidator
+from metadata.utils.entity_link import get_table_fqn
+from metadata.utils.logger import test_suite_logger
+from metadata.utils.sqa_like_column import SQALikeColumn
+
+logger = test_suite_logger()
+
+
+class BaseColumnValuesToBeNotInSetValidator(BaseTestValidator):
+    """ "Validator for column value mean to be between test case"""
+
+    def run_validation(self) -> TestCaseResult:
+        """Run validation for the given test case
+
+        Returns:
+            TestCaseResult:
+        """
+        forbidden_values = self.get_test_case_param_value(
+            self.test_case.parameterValues,  # type: ignore
+            "forbiddenValues",
+            literal_eval,
+        )
+
+        try:
+            column: Union[SQALikeColumn, Column] = self._get_column_name()
+            res = self._run_results(
+                Metrics.COUNT_IN_SET, column, values=forbidden_values
+            )
+        except (ValueError, RuntimeError) as exc:
+            msg = (
+                f"Error computing {self.test_case.name} for "
+                f"{get_table_fqn(self.test_case.entityLink.__root__)}: {exc}"
+            )
+            logger.debug(traceback.format_exc())
+            logger.warning(msg)
+            return self.get_test_case_result_object(
+                self.execution_date,
+                TestCaseStatus.Aborted,
+                msg,
+                [TestResultValue(name="countForbiddenValues", value=None)],
+            )
+
+        return self.get_test_case_result_object(
+            self.execution_date,
+            self.get_test_case_status(res == 0),
+            f"Found countInSet={res}. It should be 0",
+            [TestResultValue(name="countForbiddenValues", value=str(res))],
```

Comment:
nit: to align the code as we did in the other classes move this into `COUNT_FORBIDDEN_VALUES` constant.